package.json

```diff
@@ -140,5 +140,14 @@
     "videojs-swf": "^5.2.0",
     "watchify": "^3.6.0",
     "webpack": "^1.13.2"
+  },
+  "greenkeeper": {
+    "ignore": [
+      "browserify",
+      "qunitjs",
+      "sinon",
```

Comment:
Should we still let GK keep updating the qunit and sinon dependencies?

Reply:
Sinon will break, not sure offhand about qunit.

Reply:
updating qunit is probably fine.

Reply:
Video.js couldn't because it had to support IE8 and qunit 2 dropped support for it but contrib-hls never supported IE8.